examples/dave_demo_launch/launch/dave_integrated_demo.launch

```diff
@@ -7,12 +7,10 @@
     <arg name="set_timeout" default="false"/>
     <arg name="timeout" default="0.0"/>
     <arg name="velocity_control" default="true"/>
-    <arg name="joy_id" default="0"/>
+    <!-- <arg name="joy_id" default="0"/> -->
```

Comment:
Are these commented tags supposed to be added to this PR, or is it a mistake? I didn't see a note on these

Reply:
I will double check whether this was needed or mistake soon. Thanks!

Reply:
Added Uncommenting commit. This should be good to go.